<launch>

  <node pkg="trajectory_publisher" type="trajectory_publisher" name="trajectory_publisher" output="screen">
        <param name="trajectory_type" value="1" />
        <param name="shape_omega" value="1.2" />
        <param name="initpos_z" value="10.0" />
        <param name="reference_type" value="2" />
  </node>

</launch>
